src/components/update-reminder/update-reminder.js

```diff
@@ -27,48 +44,105 @@ const UpdateReminder = ({ versions, setDismiss }) => {
             />
           </div>
         </div>
-        <div className="update-reminder-expanded-detail">
-          <p>
-            We're excited to announce that Kedro-Viz {latest} has been released.
-            To update Kedro-Viz, copy and paste the following update command
-            into your terminal. <br />
-            <br />
-            You are currently on Kedro-Viz {installed}.
-          </p>
+        <div
+          className={classnames('update-reminder-expanded-detail', {
+            'update-reminder-expanded-detail--up-to-date': !isOutdated,
+          })}
+        >
+          {isOutdated ? (
+            <>
+              <h3>Kedro-Viz {latest} is here!</h3>
+              <p>
+                We’re excited to announce Kedro-Viz {latest} has been released.
+                To update Kedro Viz, copy and paste the following update command
+                into your terminal.{' '}
+                <a
+                  href="https://github.com/kedro-org/kedro-viz/releases"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  View full changelog.
+                </a>
+              </p>
 
-          <p className="subtext">Update command</p>
-          <div className="command-copier">
-            <CommandCopier command={command} />
-          </div>
-
-          <p className="subtext--small">
-            Refer to the release page for the set of changes introduced. <br />
-            <a
-              href="https://github.com/kedro-org/kedro-viz/releases"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              View release notes
-            </a>
-          </p>
+              <p className="subtext">Update command</p>
+              <div className="command-copier">
+                <CommandCopier command={command} isCommand />
+              </div>
+            </>
+          ) : (
+            <>
+              <h3>You're up to date</h3>
+              <p>Kedro-Viz {latest}</p>
+            </>
+          )}
+        </div>
+        <div
+          className={classnames('update-reminder-content', {
```

Comment:
hey why can this classname also have the `-expanded` in it? `'update-reminder-expanded-content'`. As it seems like the details and header follow the same classname rule